app/uk/gov/hmrc/agentuserclientdetails/AgentUserClientDetailsMain.scala

```diff
@@ -129,7 +129,7 @@ class AgentUserClientDetailsMain @Inject() (
     )
     def cleanup(): Future[Unit] = Future
       .traverse(cleanupJobs) { case (name, f) =>
-        logger.info(s"[$name] Starting work item cleanup.")
+        logger.debug(s"[$name] Starting work item cleanup.")
```

Comment:
we would have visibility in prod now with INFO (from your recent change)...these logs are useful to know the scheduler is running?